uber/decorators.py

```diff
@@ -1,28 +1,30 @@
 from uber.common import *
 
+
 def log_pageview(func):
     @wraps(func)
-    def with_check(*args,**kwargs):
-        with Session() as session:
+    def with_check(*args, **kwargs):
+        with sa.Session() as session:
```

Comment:
Whoops, just found this comment. Disregard my question from earlier. :)

That seems like a great solution - isn't this one of the reasons I had to put the prereg check inside a wrapper?